variants/fcos/okd-rpms/manifests/99-okd-master-disable-mitigations.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
I don't think these manifests are needed in `okd-rpms`, which should just contain kubelet+crio RPMs.
We do need them for `okd-machine-os`

Reply:
done